src/main/java/com/sem/btrouble/TestState.java

```diff
@@ -0,0 +1,163 @@
+package com.sem.btrouble;
+
+import com.sem.btrouble.event.PlayerEvent;
+import com.sem.btrouble.model.Drawable;
+import com.sem.btrouble.model.Game;
+import com.sem.btrouble.model.Observer;
+import com.sem.btrouble.model.Player;
+import com.sem.btrouble.model.Room;
+import com.sem.btrouble.model.Rope;
+import org.newdawn.slick.Color;
+import org.newdawn.slick.GameContainer;
+import org.newdawn.slick.Graphics;
+import org.newdawn.slick.Image;
+import org.newdawn.slick.Input;
+import org.newdawn.slick.SlickException;
+import org.newdawn.slick.TrueTypeFont;
+import org.newdawn.slick.geom.Circle;
+import org.newdawn.slick.state.BasicGameState;
+import org.newdawn.slick.state.StateBasedGame;
+import org.newdawn.slick.util.ResourceLoader;
+
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Test state
+ */
+public class TestState extends BasicGameState implements Observer {
+    private TrueTypeFont font;
+    private List<Drawable> drawables;
+    private Image background;
+    private Game game;
+    private Player player;
+
+    /**
+     * Initialize method of the slick2d library.
+     *
+     * @param gc
+     *            should be the GameContainer containing the game.
+     * @param sbg
+     *            the reference to the StateBasedGame.
+     * @throws SlickException
+     *             when the game could not be initialized.
+     */
+    @Override
+    public void init(GameContainer gc, StateBasedGame sbg) throws SlickException {
+        // load font from a .ttf file
+        try {
+            InputStream inputStream = ResourceLoader.getResourceAsStream("Sprites/IndieFlower.ttf");
+            java.awt.Font awtFont = java.awt.Font.createFont(java.awt.Font.TRUETYPE_FONT,
+                    inputStream);
+            awtFont = awtFont.deriveFont(24f); // set font size
+            font = new TrueTypeFont(awtFont, false);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        drawables = new ArrayList<Drawable>();
```

Comment:
List is de interface. ArrayList is een implementatie van de List. Je zou hier drawables ook kunnen initialiseren met een LinkedList oid. Door List te gebruiken, blijft het wat flexibeler.